src/render/VolumeRaycastShader.cpp

```diff
@@ -115,7 +116,7 @@ VolumeRaycastShader::VolumeRaycastShader() : m_use_blending{ false }, m_blend_vo
     //To ensure we update dataPos and t_hit to reflect a ray from entry point to exit
     "dataPos = camPos + t_hit.x * geomDir;\n"
     "t_hit.y = t_hit.y-t_hit.x; \n"
-    "t_hit.x = 0.0f; \n"
+    "t_hit.x = 0.0001f; \n"
```

Comment:
seems like you're offsetting a lot of numbers slightly - I'd add a comment somewhere towards the top of this block explaining why

Reply:
I added a comment about this line